doc_src/index.hdr.in

```diff
@@ -991,15 +991,19 @@ Some bindings are shared between emacs- and vi-mode because they aren't text edi
 
 - @key{Control,X} copies the current buffer to the system's clipboard, @key{Control,V} inserts the clipboard contents.
 
-- @key{Alt,D} moves the next word to the <a href="#killring">killring</a>.
+- @key{Alt,d} moves the next word to the <a href="#killring">killring</a>.
 
-- @key{Alt,H} (or @key{F1}) shows the manual page for the current command, if one exists.
+- @key{Alt,h} (or @key{F1}) shows the manual page for the current command, if one exists.
 
-- @key{Alt,L} lists the contents of the current directory, unless the cursor is over a directory argument, in which case the contents of that directory will be listed.
+- @key{Alt,l} lists the contents of the current directory, unless the cursor is over a directory argument, in which case the contents of that directory will be listed.
 
-- @key{Alt,P} adds the string '`| less;`' to the end of the job under the cursor. The result is that the output of the command will be paged.
+- @key{Alt,p} adds the string '`| less;`' to the end of the job under the cursor. The result is that the output of the command will be paged.
 
-- @key{Alt,W} prints a short description of the command under the cursor.
+- @key{Alt,w} prints a short description of the command under the cursor.
+
+- @key{Alt,e} Edit the current command line in an external editor defined by the `$EDITOR` env var, else vim if it is in `$PATH`, else emacs if it is in `$PATH`, else an error is displayed.
```

Comment:
Suggest: `@key{Alt,e} edit [lowercase] the current command line in an external editor. The editor is chosen from the first available of the `$EDITOR` variable, the `$VISUAL` variable [if we also add this], or the `vim` or `emacs` commands.